core-galleon-pack/wildfly-feature-pack-build.xml

```diff
@@ -0,0 +1,126 @@
+
+            <!--
+             This one is here to avoid
+             ERROR [org.jboss.as.domain.http.api.undertow] (MSC service thread 1-4) WFLYDMHTTP0004: Unable to load error context for slot main, disabling error context.
+              -->
+            <package name="org.jboss.as.domain-http-error-context.main"/>
+
+            <!-- cleanup runtime dirs -->
+            <package name="cleanup.standalone.config.history.dir"/>
+            <package name="cleanup.standalone.log.dir"/>
+            <package name="cleanup.standalone.data.dir"/>
+            <package name="cleanup.standalone.tmp.vfs"/>
+        </packages>
+    </config>
+
+    <config model="domain">
+        <props>
+            <prop name="config.branch-is-batch" value="true"/>
+            <prop name="--empty-domain-config" value=""/>
+            <prop name="--remove-existing-domain-config" value=""/>
+            <prop name="--empty-host-config" value=""/>
+            <prop name="--remove-existing-host-config" value=""/>
+        </props>
+        <packages>
+            <package name="product.conf" optional="true"/>
+            <package name="misc.domain"/>
+            <package name="org.jboss.as.domain-management.main"/>
+            <package name="org.jboss.as.domain-http-error-context.main"/>
+
+            <!-- cleanup runtime dirs -->
+            <package name="cleanup.domain.config.history.dir"/>
+            <package name="cleanup.domain.servers.dir"/>
+            <package name="cleanup.domain.log.dir"/>
+            <package name="cleanup.domain.data.dir"/>
+        </packages>
+    </config>
+
+    <config model="host">
+        <config-deps>
+            <config-dep id="domain-dep" model="domain"/>
+        </config-deps>
+        <props>
+            <prop name="config.branch-is-batch" value="true"/>
+            <prop name="--domain-config" value="domain.xml"/>
+            <prop name="--empty-host-config" value=""/>
+            <prop name="--remove-existing-host-config" value=""/>
+        </props>
+    </config>
+
+    <config name="standalone.xml" model="standalone">
+        <!-- config name is the resulting xml file name which can be changed by setting the property below
+        <props>
+            <prop name="- -server-config" value="standalone-custom.xml"/>
+        </props>
+        -->
+        <feature-group name="standalone"/>
+    </config>
+
+    <config name="domain.xml" model="domain">
+        <!-- config name is the resulting xml file name which can be changed by setting the property below
+        <props>
+            <prop name="- -domain-config" value="domain-custom.xml"/>
+        </props>
+        -->
+        <feature-group name="domain"/>
+    </config>
+
+    <config name="host.xml" model="host">
+        <!-- config name is the resulting xml file name which can be changed by setting the property below
+        <props>
+            <prop name="- -host-config" value="host-custom.xml"/>
```

Comment:
Should that space in --host-config be there given this is commented out?

Reply:
Unfortunately, yes, -- is not allowed in the comments.